targets/gnome

```diff
@@ -10,7 +10,7 @@ CHROOTBIN='crouton-noroot startgnome gnome-session-wrapper'
 
 ### Append to prepare.sh:
 install --minimal evolution-data-server gnome-control-center \
-        gnome-screensaver gnome-session gnome-session-fallback \
+        gnome-screensaver gnome-session gnome-session-flashback \
```

Comment:
`gnome-session-flashback` doesn't exist prior to trusty or in wheezy/kali, so you need to do: `gnome-session-flashback,ubuntu~precise+ubuntu~quantal+ubuntu~raring+ubuntu~saucy+debian~wheezy+kali=gnome-session-fallback`....which really isn't the best syntax.

Alternatively, you could set a variable with the package name ahead of time, and use `if release -lt trusty -lt wheezy -eq kali` to see when fallback is necessary.